### PR TITLE
Added rule for bbk.bund.de.

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

**Comment by J0WI** [comment]
We already have some rules containing `bund.de`. Maybe you can document the relationship between them or merge the files?

**Comment by Jochen-A-Fuerbacher** [comment]
Merging those files wouldn't be a good idea, in my opinion. Most of them handle some websites of federal offices of Germany, which doesn't have anything in common.

I could put a comment to the top of this (bbk.bund.de) file, explaining that there are other files for the bund.de domain. Would that be ok?

**Comment by J0WI** [comment]
Yes, this would be helpful. We already have such references in big rulesets like Amazon.xml or Google.xml.

File: src/chrome/content/rules/bbk.bund.de.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+-->
+<ruleset name="Bundesamt für Bevölkerungsschutz und Katastrophenhilfe">
+	<!-- Federal Office of Civil protection and Disaster Assistance. -->
+
+	<target host="bbk.bund.de" />
+	<target host="www.bbk.bund.de" />
+	<target host="austausch.bbk.bund.de" />
+	<target host="warnung.bund.de" />
+	
+	<rule from="^http://bbk\.bund\.de/" to="https://www.bbk.bund.de/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/bbk.bund.de.xml]: Please document the mismatch.

File: src/chrome/content/rules/bbk.bund.de.xml [hunk 2]
```diff
@@ -1,4 +1,11 @@
 <!--
+	For other bund.de coverages, see:
+ 	- Federal-Office-for-Information-Security.xml
+	- Bundesnachrichtendienst.xml
+	- Bundesministerium_des_Innern.xml
+	- Federal_Office_of_Administration.xml
+	- Verwaltung_Online.xml
```
Comment (J0WI) [inline src/chrome/content/rules/bbk.bund.de.xml]: We use to make a reference from the referenced files back to this one. See examples in `Google.xml` or `Amazon.xml`.

**Comment by J0WI** [comment]
In such cases we use to define one 'main ruleset' where all references are listed and then just refer to the list in that main rule. Otherwise we have to maintain the list of references in all related rulesets. See examples in `Google.xml` or `Amazon.xml` (and the referred rules there).
I don't know if there is one ruleset of federal offices that could be defined as 'main rule'?

Some of the related rules do not fit our current requirements and break our tests. Could you just fix the errors there? Maybe they need to be rewritten, but this is likely a separate PR.

**Comment by Jochen-A-Fuerbacher** [comment]
The `Verwaltung_Online.xml` ruleset contains the main `bund.de` domain. So I put the list there.

I will look at the rules that break the tests and open an other PR for that.

**Comment by Jochen-A-Fuerbacher** [comment]
Fixed rules to pass tests in PR #8625.

**Comment by Jochen-A-Fuerbacher** [comment]
Is there anything still to do here?

**Comment by J0WI** [comment]
I would like to have all errors fixed before merging. #8626 needs to be reviewed by somebody else. There are still some errors in the fetch test, but they can be fixed here, when this branch gets rebased.

**Comment by J0WI** [comment]
@Jochen-A-Fuerbacher all dependencies are merged. Could you please rebase your changes and make sure, that all tests are passing?

**Comment by Jochen-A-Fuerbacher** [comment]
There are now two subdomains in `Federal_Office_of_Administration.xml` that break tests. See #8853.

**Pull request merged**